src/main/kotlin/parts/wisdom/arcraider/visualization/SwingApp.kt

```diff
@@ -1,22 +1,66 @@
 package parts.wisdom.arcraider.visualization
 
+import parts.wisdom.arcraider.Task
 import parts.wisdom.arcraider.loadTaskFromFile
+import java.awt.Dimension
 import java.awt.EventQueue
+import java.awt.FlowLayout
 import java.io.File
+import javax.swing.*
 
 private const val WINDOW_HEIGHT_PIXEL = 600
 private const val WINDOW_WIDTH_PIXEL = 800
 
+private var currentGridPanel : GridPane? = null
+
 private fun createAndShowGUI() {
     val frame = ARCWindow(WINDOW_WIDTH_PIXEL, WINDOW_HEIGHT_PIXEL, "ARCRaider")
     val task = loadTaskFromFile(
             File(object {}.javaClass.classLoader.getResource("extend_right.json").toURI()))
```

Comment:
Now that we have the ability to load any file, should we still be trying to find this specific one? Maybe just load an empty grid by default now?

Reply:
I dunno seems fine to load something by default